pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -446,5 +447,27 @@ private synchronized void deleteDynamicConfigurationOnZk(String configName) {
     public String version() throws Exception {
         return PulsarVersion.getVersion();
     }
+
+    @POST
+    @Path("/shutdown")
+    @ApiOperation(value =
+            "Shutdown broker gracefully.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 204, message = "Execute shutdown command successfully"),
+            @ApiResponse(code = 403, message = "You don't have admin permission to update service-configuration"),
+            @ApiResponse(code = 500, message = "Internal server error")})
+    public void shutDownBrokerGracefully(
+            @QueryParam("maxConcurrentUnloadPerSec") int maxConcurrentUnloadPerSec,
```

Comment:
Better to add a description here, if the value absent(value=0) means no concurrent limitation.